docs/integrations/saas-cloud/proofpoint-on-demand.md

```diff
@@ -0,0 +1,187 @@
+---
+id: proofpoint-on-demand
+title: Proofpoint On Demand
+sidebar_label: Proofpoint On Demand
+description: The Proofpoint On Demand app for Sumo Logic provides an improved security posture by analyzing message logs.
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/proofpoint-logo.png')} alt="proofpoint-logo" width="100"/>
+
+Proofpoint On Demand is a cloud-based cybersecurity platform that offers a wide range of services to protect businesses against cyber threats. This includes email security, threat intelligence, information protection, and compliance solutions. The Proofpoint on Demand app for Sumo Logic enables insights into the functioning of email security policies and helps you gain visibility into the effectiveness of email security policies, therefore making informed decisions to improve security posture. This app will further provide information about the encryption status of the communication.
+
+Key features of the Proofpoint On Demand app include: 
```

Comment:
```suggestion
Key features of the Proofpoint on Demand app include: 
```

Reply:
@JV0812 you marked resolved, but it isn't fixed - please click "commit suggestion"